src/plugins/administrative_templates/registry/polregistrysource.cpp

```diff
@@ -144,8 +144,6 @@ void PolRegistrySource::markValueForDeletion(const std::string &key, const std::
 
     std::string deleteValueName = "**del." + valueName;
 
-    clearValue(key, valueName);
```

Comment:
This should not be deleted as we cannot replace value of the entry, so first we need to delete it and than add new entry but with a different type.